src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -1,48 +1,172 @@
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all MythicSpawners.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "spawners", (attribute, object) -> {
+                return new ListTag(getSpawnerManager().getSpawners(), MythicSpawnerTag::new);
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.damage_modifiers[<mob_id>]>
+            // @returns MapTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a map of the damage modifiers for a MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(MapTag.class, ElementTag.class, "damage_modifiers", (attribute, object, id) -> {
+                MythicMob mob = getMythicMob(id.asString());
+                if (mob == null) {
+                    return null;
+                }
+                return new MapTag(mob.getDamageModifiers().entrySet().stream().map((e) -> Map.entry(new StringHolder(e.getKey()), new ElementTag(e.getValue()))).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_path[<mob_id>]>
+            // @returns ElementTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns the path to the configuration file for the MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(ElementTag.class, ElementTag.class, "mob_path", (attribute, object, id) -> {
+                MythicMob mob = getMythicMob(id.asString());
+                if (mob == null) {
+                    return null;
+                }
+                return new ElementTag(mob.getConfig().getFile().getPath(), true);
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.item_path[<item_id>]>
+            // @returns ElementTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns the path to the configuration file for the MythicItem from an Item ID.
+            // -->
+            tagProcessor.registerTag(ElementTag.class, ElementTag.class, "item_path", (attribute, object, id) -> {
+                Optional<MythicItem> item = getItemManager().getItem(id.asString());
+                return item.map(mythicItem -> new ElementTag(mythicItem.getConfig().getFile().getPath(), true)).orElse(null);
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.packs>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a list of all Mythic pack IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "packs", (attribute, object) -> new ListTag(MythicBukkit.inst().getPackManager().getPacks(), (pack) -> new ElementTag(pack.getName())));
```

Comment:
Should still be on a new line (`-> {`).
Also, should make these `ElementTag`s plain-text